src/iocore/net/OCSPStapling.cc

```diff
@@ -53,6 +53,9 @@ extern ClassAllocator<FetchSM> FetchSMAllocator;
 // clang-format off
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Wunused-function"
+#ifndef GENERAL_NAME_it
+DECLARE_ASN1_ITEM(GENERAL_NAME)
+#endif
```

Comment:
`GENERAL_NAME_it` in openssl is declared in the expansion of this macro:
https://github.com/openssl/openssl/blob/806bbafe2df5b699feac6ef26e50c14e701950cf/include/openssl/x509v3.h.in#L530
For some reason, boringssl has removed most of the contents of x509v3.h, moving some (but not all of it) to x509.h .

See:  https://github.com/openssl/openssl/blob/806bbafe2df5b699feac6ef26e50c14e701950cf/include/openssl/asn1.h.in#L244

Reply:
Right, they (BoringSSL) assumed it's only used internally and move it into a private header.

Reply:
OK hopefully this will be temporary.  Should we raise an issue in BoringSSL? (Is that even possible?)